src/app/applications/components/application-resources-tree/application-resources-tree.tsx

```diff
@@ -142,6 +153,9 @@ export const ApplicationResourcesTree = (props: {
                         <div className='application-resources-tree__node-labels'>
                             {props.nodeLabels && props.nodeLabels(node).map((label) => <span key={label}>{label}</span>)}
                             <span>{kubeState.kind}</span>
+                            {kubeState.status && kubeState.status.containerStatuses && (
```

Comment:
We already have code which creates labels for pods. It is good to keep code with similar responsibility in one please. Please move your changes to https://github.com/argoproj/argo-cd-ui/blob/f9c39fbc3b2e8c64fe19dd74593d8dcb1e439cc9/src/app/applications/components/application-details/application-details.tsx#L352